### PR TITLE
using middleware

### PR DESCRIPTION
**Pull request by Marialaina**

### CODE REVIEW

**Comment by Marialaina** [comment]
by the way I named the file workout1 just in case you wanted to use Adewale's schema

File: models/Workout1.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+//CONNECTING TO MY MODEL DATABASE
+require("dotenv").config();
+
+const {Schema, model} = require("../db/connection")
+
+//CREATING MY TURTLE SCHEMA
```
Comment (dezerey11) [inline models/Workout1.js]: replace turtle with workout

File: models/Workout1.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+const WorkoutSchema = new Schema({
+    title: String,
+    text: String,
+    date: String
+});
+
+const Workout = model('Workout', WorkoutSchema);
+// console.log("/////////////////");
+// console.log(Turtle);
```
Comment (dezerey11) [inline models/Workout1.js]: Do we need these console.logs?  Maybe remove this

File: package.json [hunk 3]
```diff
@@ -11,6 +11,7 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "config": "^3.3.6",
```
Comment (dezerey11) [inline package.json]: Are we using this anywhere? I'm not sure if we need it

Comment (Marialaina) [reply package.json]: I was getting an error so I installed it when I was troubleshooting

**Review by dezerey11 — APPROVED** [review]

**Pull request merged**